File: official/guides/getting-started/golang/create_from_address.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
    apiKey := os.Getenv("<YOUR_TEST/PRODUCTION_API_KEY>")
    client := easypost.New(apiKey)

    fromAddress, err := client.CreateAddress(
        &easypost.Address{
            Company: "EasyPost",
            Street1: "417 Montgomery Street",
            Street2: "5th Floor",
            City:    "San Francisco",
            State:   "CA",
            Zip:     "94104",
            Phone:   "415-528-7555",
        },
        &easypost.CreateAddressOptions{},
    )
}

File: official/guides/getting-started/node/create_from_address.js
const Easypost = require('@easypost/api');
const api = new Easypost('<YOUR_TEST/PRODUCTION_API_KEY>');

const fromAddress = new api.Address({
  company: 'EasyPost',
  street1: '417 Montgomery Street',
  street2: '5th Floor',
  city: 'San Francisco',
  state: 'CA',
  zip: '94104',
  phone: '415-528-7555',
});

fromAddress.save().then(console.log);
